source/includes/cli-reference/_commands.html.md

```diff
@@ -135,13 +135,16 @@ Usage: bonsai log [OPTIONS]
 Options:
   --brain TEXT    Override to target another BRAIN.
   --project TEXT  Override to target another project directory.
+  --follow        Continually follow simulator's output
```

Comment:
Just one nit (that I mentioned in the Diff as well. Help messages should have punctuation.